torch/csrc/lazy/core/lazy_graph_executor.cpp

```diff
@@ -998,7 +1002,7 @@ std::vector<at::Tensor> LazyGraphExecutor::GetTensorsFused(
   SyncTensorsConfig config;
   config.force_ltc_data = false;
   auto async = SyncTensorsGraphInternal(tensors, {}, config);
-  if (async != nullptr) {
+  if (FLAGS_torch_lazy_use_thread_pool && async != nullptr) {
```

Comment:
Did you test how to set this flag at runtime so that XLA folks would be able to set it easily during their migration? 

I think there is some way to import caffe2.workspace and set flags like this.  @Krovatkin may be the one that showed me before.

Reply:
Are we building with gflags by default?

Would be interested to know how this can be turned on/off in a script as well

Reply:
I can add a python binding for setting it.